sysutils/peco/Portfile

```diff
@@ -70,63 +34,40 @@ go.vendors-append   github.com/davecgh/go-spew    346938d642f2ec3594ed81d8744619
                     github.com/pkg/errors         248dadf4e9068a0b3e79f02ed0a610d935de5302 \
                     github.com/stretchr/testify   18a02ba4a312f95da08ff4cfc0055750ce50ae9e
 
-checksums-append    davecgh-go-spew-${go-spew.version}.tar.gz \
+checksums-append    ${go-spew.distfile} \
                         md5 40113f77febc1bdf8207c2c8c9f67293 \
                         sha1 b02ca0e6655fe8db4658ac265df87ae8cb52a44a \
```

Comment:
Thanks for the feedback. It's valuable to get this feedback now, while there aren't many ports using the portrgoup yet.

> Out of curiosity: why are there md5 and sha1 checksums, but no rmd160 and size?

I didn't notice that; you'd have to ask @kimuraw. My only goal for this portfile was to adapt the existing port to use the portgroup. The portgroup has no special requirements about the checksums.

> it could be useful to instead specify something like ...
> Or perhaps some syntax that's more similar to checksums-append which would include all the data needed to download the packages, but also all the checksums. Basically merging the two tables.

We could do that. I find your first version to be pretty hard to understand, but the second one is nice. My first thought is that I wouldn't want to lock down the supported checksum types, but I guess it should be possible to be flexible.

Reply:
I have an implementation of your second suggestion here: https://github.com/amake/macports-ports/commit/b3116f301a8e5f311cbd82bd5a465b5d2bd2c2e8#diff-5fa707373a52170a3e130a05fbbe1d0b

If you don't have any more thoughts on this particular item, I will push it to master.

Reply:
Wonderful. I like the code you are linking, and yes, I agree that there's no need to rush with all the changes at the same time. The rest seems fine to me.

Reply:
> I didn't notice that; you'd have to ask @kimuraw. My only goal for this portfile was to adapt the existing port to use the portgroup. The portgroup has no special requirements about the checksums.

Yah, I'd fix the checksums...